pages/scripts/delayed.js

```diff
@@ -86,47 +84,32 @@ async function OptanonWrapper() {
     localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(adobeSettings));
   }
 
-  clearDataLayer();
-  loadScript(`https://assets.adobedtm.com/d17bac9530d5/a14f7717d75d/launch-aa66aad171be${isProd ? '.min' : ''}.js`, () => {
-    pushOneTrustConsentGroups();
-
-    loadScript('https://cdns.us1.gigya.com/js/gigya.js?apikey=3_IscKmAoYcuwP8zpTnatC3hXBUm8rPuI-Hg_cZJ-jL-M7LgqCkxmwe-ps1Qy7PoWd', () => {
```

Comment:
I think this script might need to stay. I think it should be moved out the adobe loadscript